data/distro.yml

```diff
@@ -21,6 +21,7 @@
         <pre><code>
           <span class="unselectable">$</span> sudo apt install gnome-software-plugin-flatpak
         </code></pre>
+        <p>Note: graphical installation of Flatpak apps may not be possible with Ubuntu 20.04.</p>
```

Comment:
I am not sure if this text is ideal. Maybe something like `Note: graphical installation of Flatpak apps is not officially supported on Ubuntu 20.04 and doing this step is not recommended there.` would be more appropriate.

Reply:
Maybe it's worth explaining why? Say "because it results in both Snap Store and GNOME Software being installed side by side".

Reply:
Done. :-) Let me know if it is ok now. Thanks!